mne/viz/topomap.py

```diff
@@ -1389,9 +1389,12 @@ def plot_evoked_topomap(evoked, times="auto", ch_type=None, layout=None,
     vmin = np.min(vlims)
     vmax = np.max(vlims)
     if cmap == 'interactive':
-        cmap = (None, True)
+        cmap = ('Reds' if vmin >= 0. else 'RdBu_r', True)
     elif not isinstance(cmap, tuple):
+        if cmap is None:
+            cmap = 'Reds' if vmin >= 0. else 'RdBu_r'
         cmap = (cmap, False if len(times) > 2 else True)
```

Comment:
could we avoid having these hard coded default cmaps in different places in the code?